code/modules/mob/living/simple_animal/abnormality/teth/pale_horse.dm

```diff
@@ -0,0 +1,221 @@
+#define STATUS_EFFECT_MORTIS /datum/status_effect/mortis
+// Coded by Coxswain
+/mob/living/simple_animal/hostile/abnormality/pale_horse
+	name = "Pale Horse"
+	desc = "The riderless pale horse of the apocalypse."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "palehorse"
+	icon_living = "palehorse"
+	icon_dead = "palehorse"
+	speak_emote = list("neighs")
+	threat_level = TETH_LEVEL
+	maxHealth = 800
+	health = 800
+	pixel_x = -16
+	base_pixel_x = -16
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.5, WHITE_DAMAGE = 1.5, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 0)
```

Comment:
The one guy with daredevil:
![image](https://user-images.githubusercontent.com/61567407/232250098-3c14248c-0882-496a-99ed-b467f13c1f76.png)

Reply:
Would 0.1 be better? It's really not a tough abnormality...

Reply:
Nah this is just a funny comment, not a change requested.